Makefile

```diff
@@ -88,8 +88,13 @@ quicktype-elm: 				deps-quicktype-elm 				gen-quicktype-elm 				test-quicktype-e
 
 # Prerequisite verification
 verify-prereq-generator:
+ifdef WINDOWS
```

Comment:
Let's see if we can use some makefile primitives to detect windows automatically, this is the first google result for this: https://gist.github.com/sighingnow/deee806603ec9274fd47

Reply:
Done. I was experimenting with this originally and hitting some issues but now it seems to work perfectly.